### ISSUE TITLE
Validation script emits bogus suggestion regarding 'swissprot_identifier'

### ISSUE THREAD

**Issue by fedde-s** [cBioPortal/cbioportal#2628]
When the 'swissprot_identifier' field is missing in the metadata file for a mutation profile, the study validation script suggests adding a MAF column that is not documented and will not affect the issue, instead of explaining the actual issue:

```
WARNING: data_mutations_extended.txt: column 73: A SWISSPROT column was found without an associated 'swissprot_identifier' annotation column; assuming 'swissprot_identifier' column contains Swiss-Prot name.
```

**Comment by fedde-s** [comment]
Fixed in #2629.

**Comment by sandertan** [comment]
Merged in 1.7.3
